crates/burn-tensor/src/tests/ops/mod.rs

```diff
@@ -7,6 +7,13 @@ mod arange;
 mod arange_step;
 mod arg;
 mod argwhere_nonzero;
+mod bitwise_and;
+mod bitwise_and_scalar;
+mod bitwise_not;
+mod bitwise_or;
+mod bitwise_or_scalar;
+mod bitwise_xor;
+mod bitwise_xor_scalar;
```

Comment:
I think tests for bitwise operators could be grouped into a single file 🙂

Reply:
Done